.github/workflows/scripts/get_dependencies.sh

```diff
@@ -119,6 +119,7 @@ get_gcc() {
 # Usage:
 #   get_gcovr
 get_gcovr() {
+  sudo apt-get install libxml2-dev libxslt-dev python-dev
```

Comment:
add a `sudo apt update` as the first command just in case for some reason a project has no other `apt` dependencies.